### ISSUE TITLE
Any toggle creation

### ISSUE THREAD

**Issue by y0ur1kopf** [anytypeio/roadmap#66]
**Relevant Request:**
• [Make "toggle" a feature of every block](https://community.anytype.io/t/make-toggle-a-feature-of-every-block/1001)

**Comment by rushthomas** [comment]
Is there a description of Feature Request that describes what this is in reference to?

**Comment by jmetrikat** [comment]
hey @rushthomas, this should best describe it: https://community.anytype.io/t/make-toggle-a-feature-of-every-block/1001

**Comment by jwierzbo** [comment]
@y0ur1kopf @Seanytype  Any chance of getting prioritised on this? Even Apple Notes has that issue with the latest version…

**Comment by jwierzbo** [comment]
@y0ur1kopf @Seanytype 

It is not such a complex feature that it requires many sprints, but it is really a blocker for writing long notes. Going through such a long document is a nightmare, especially on the phone.

Please allow us to collapse the headers, at least.

The other competitors (Notion, Obsidian, Craft, AppFlowy, AFFiNE, Apple Notes) have it...

**Comment by Alex-C-github** [comment]
I agree with @jwierzbo, collapsible headings would be highly appreciated.

**Comment by blooddiamondkim** [comment]
> [@y0ur1kopf](https://github.com/y0ur1kopf) [@Seanytype](https://github.com/Seanytype)
> 
> It is not such a complex feature that it requires many sprints, but it is really a blocker for writing long notes. Going through such a long document is a nightmare, especially on the phone.
> 
> Please allow us to collapse the headers, at least.
> 
> The other competitors (Notion, Obsidian, Craft, AppFlowy, AFFiNE, Apple Notes) have it...

Throwing in my vote. I'm trying to move from Notion because I want something more privacy focused that I can self host, and AnyType is fantastic, but the lack of toggles for organizing has been a big mental blocker for me. Would love to see it implemented.

**Comment by Teronbullock** [comment]
I'm all in on this app; the team has done a wonderful job. I would also like to vote for Toggle Headers. This feat is great when dealing with large coding doc notes.

**Comment by poweredupbass** [comment]
YESSS ALL the above! This would be incredibly helpful for vertical space management with appropriate heading cases for my formatting tics. Excited for this one!!

**Comment by jwierzbo** [comment]
@y0ur1kopf has been a year since it was moved to the backlog... Any change to move it to the sprint scope?

**Comment by Mahi** [comment]
This is the most needed feature by far, the only thing really holding people back from transitioning from Notion to AnyType.

**Comment by Mahi** [comment]
This is by far the most needed feature for Anytype right now, it's the only thing holding my friend group from transitioning away from Notion.

**Comment by 3m0W33D** [comment]
Most needed feature for me to swap to this software literally. I have so many times I take notes and it becomes way too long toggling certain parts make it so much more readable.

**Comment by noblesoul7** [comment]
When does a backlogged feature … get unbacklogged? 👀

**Comment by u-br** [comment]
**Suggestion: Add toggle/untoggle to the block's menu**, next to delete/copy/paste/duplicate/style etc.
**Why:** At toggling/untoggling, it would be great if an unfolded toggle-block could be folded from anywhere in that block. 
E.g. if there's a long unfolded toggle with lots of content, one needs to scroll all the way up first before one can click that tiny little ">" to fold the block.
**How**: The vertical block menu is always visible and accessible, regardless if the block is folded or unfolded. Clicking ⇧⌘T is only possible in the "header" block that's toggling. Clicking the vertical block menu is possible from anywhere.

**Comment by poweredupbass** [comment]
This ^^^ much needed. And toggle should be applicable to all font/header sizes. Titles down thru headers to text. Please and please 🙏

Also need to be able to open a toggle and add into that toggled block without having to use the mouse to select that line. Right Arrow key to toggle/untoggle it then Enter/Return to go to the internal block line. I hate having to move away from home row just to select the interior of a toggle.
